app/Http/Controllers/TaskController.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Task;
+use Illuminate\Http\Request;
+use Illuminate\Validation\Rule;
+
+class TaskController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $tasks = Task::orderBy('created_at', 'asc')->get();
```

Comment:
No query logic in Controller, it should be done in Model

Reply:
move the above logic to model